Source: oahong<at>gmail.com
Upstream: ignores my emails.
Reason: Fix make distcheck, add zhengma support.
Index: src/Makefile.am
===================================================================
--- src/Makefile.am	(版本 250)
+++ src/Makefile.am	(工作副本)
@@ -1,4 +1,3 @@
 DEFS = -DPKGDATADIR=\"$(pkgdatadir)\"
 
-INCLUDES = -I. -I$(top_srcdir)/lib -I$(top_srcdir)/xpm
-AM_CFLAGS = -O2 -fno-strength-reduce
+AM_CPPFLAGS = -I. -I$(top_srcdir)/lib -I$(top_srcdir)/xpm 
Index: tools/Makefile.am
===================================================================
--- tools/Makefile.am	(版本 250)
+++ tools/Makefile.am	(工作副本)
@@ -1,7 +1,7 @@
 DEFS = -DPKGDATADIR=\"$(pkgdatadir)\"
 
 toolsdir = $(pkgdatadir)/tools
-INCLUDES = -I$(top_srcdir)/src -I$(top_srcdir)/lib
+AM_CPPFLAGS = -I$(top_srcdir)/src -I$(top_srcdir)/lib
 tools_DATA = winmb2fcitx.py
 EXTRA_DIST = $(tools_DATA)
 
Index: doc/Makefile.am
===================================================================
--- doc/Makefile.am	(版本 250)
+++ doc/Makefile.am	(工作副本)
@@ -1,5 +1,4 @@
-docdir = $(pkgdatadir)/doc
-doc_DATA = \
-        cjkvinput.txt wb_fh.htm fcitx3.pdf pinyin.txt fcitx_trunk.doxyfile \
+dist_doc_DATA = \
+        cjkvinput.txt wb_fh.htm fcitx3.pdf pinyin.txt \
 	fcitx3.odt Develop_Readme.GBK API.txt
-EXTRA_DIST = $(doc_DATA)
+EXTRA_DIST = fcitx_trunk.doxyfile
Index: data/Makefile.am
===================================================================
--- data/Makefile.am	(版本 254)
+++ data/Makefile.am	(工作副本)
@@ -1,13 +1,13 @@
 tabledir = $(pkgdatadir)/data
-table_DATA = cj.mb erbi.mb punc.mb pybase.mb pyphrase.mb pySym.mb sp.dat \
-	     wbx.mb wanfeng.mb wbpy.mb qxm.mb zrm.mb db.mb \
-	     tables.conf vk.conf gbks2t.tab config AutoEng.dat \
-             pyPhrase.org qxm.txt zrm.txt erbi.txt wbpy.txt gbkpy.org db.txt \
-             wanfeng.txt cj.txt wbx.txt
-EXTRA_DIST = $(table_DATA)
+mb_extra = punc.mb pySym.mb sp.dat tables.conf vk.conf gbks2t.tab config AutoEng.dat
+mb_data = cj.mb erbi.mb wanfeng.mb wbpy.mb qxm.mb db.mb wbx.mb zrm.mb zhengma.mb \
+		pybase.mb pyphrase.mb
+table_DATA = $(mb_data) $(mb_extra)
+table_SOURCE = pyPhrase.org qxm.txt zrm.txt erbi.txt wbpy.txt gbkpy.org db.txt \
+             wanfeng.txt cj.txt wbx.txt zhengma.txt
+EXTRA_DIST = $(table_SOURCE) $(mb_extra)
+DISTCLEANFILES = $(mb_data) $(py_data)
 
-mb_data = cj.mb erbi.mb wanfeng.mb wbpy.mb qxm.mb db.mb wbx.mb zrm.mb
-
 %.mb: %.txt
 	../tools/txt2mb $< $@
 
@@ -21,4 +21,3 @@
 all: tools $(mb_data) $(py_data)
 	rm -f pyERROR
 	rm -f pyPhrase.ok
-	
Index: xpm/Makefile.am
===================================================================
--- xpm/Makefile.am	(版本 294)
+++ xpm/Makefile.am	(工作副本)
@@ -1,9 +1,8 @@
 xpmdir = $(pkgdatadir)/xpm
-xpm_DATA = about_icon.xpm chnPunc.xpm engPunc.xpm ft-0.xpm ft-1.xpm \
+EXTRA_DIST = about_icon.xpm chnPunc.xpm engPunc.xpm ft-0.xpm ft-1.xpm \
 	fullcorner.xpm gbk-0.xpm gbk-1.xpm halfcorner.xpm lock-0.xpm \
 	lock-1.xpm logo.xpm lx-0.xpm lx-1.xpm next.xpm prev.xpm vk-1.xpm vk.xpm \
-	full-letter.png half-letter.png full-punct.png half-punct.png \
-    Ziranma.png PinYin.png Wubi.png Erbi.png CangJie.png \
-    Logo.png \
+	inactive.xpm active.xpm
+xpm_DATA_DIST =	full-letter.png half-letter.png full-punct.png half-punct.png \
+	Ziranma.png PinYin.png Wubi.png Erbi.png CangJie.png Logo.png \
 	Jian.png Fan.png Zhong.png Ying.png Lian.png WuLian.png gbk.png gb.png 
-EXTRA_DIST = $(xpm_DATA)
